### PR TITLE
ECF-1-487: Updates to the FIP journey for induction tutors

### PR DESCRIPTION
**Pull request by amazimbe**
### Context
Added page titles
Updated content
Added a new “speed bump” page screen for “FIP Cohort: Choose the type of training"
Removed the “FIP Cohort: estimated number of participants” page
Removed the “Read and accept privacy and data policy” page

Link to designs: https://miro.com/app/board/o9J_ldVNkCY=/?moveToWidget=3074457355778670391&cot=14
Link to content: https://docs.google.com/document/d/11ubyUyMMz7OSF1Tu1JT6Ehg38l31vOfT-hZUcUTOFOM/edit#heading=h.6my5w2kg1gyb

### CODE REVIEW

**Comment by amazimbe** [comment]
@callumacrae  how do the percy changes get approved?

**Review by kicferk1 — APPROVED** [review]

File: app/views/schools/cohorts/add_participants.html.erb [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
   <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-m">2021</span>
-    <h1 class="govuk-heading-l">Add teachers and mentors</h1>
-
-    <p class="govuk-body">From <strong>July</strong>, you can confirm which early career teachers and mentors are starting a programme this year.</p>
-    <p class="govuk-body">We will let you know when you can get started.</p>
-
-    <p class="govuk-body">For each person, you will need:</p>
-
+    <span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
```
Comment (ethanmills) [inline app/views/schools/cohorts/add_participants.html.erb:10]: Let's use `@cohort.display_name` here. They're the same at the moment, but they have changed before

File: app/views/schools/cohorts/programme_choice.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<% content_for :title, "Your induction programme" %>
+
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: schools_cohort_path(@cohort.start_year))
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
```
Comment (ethanmills) [inline app/views/schools/cohorts/programme_choice.html.erb:10]: ditto

File: app/views/schools/cohorts/programme_choice.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <h1 class="govuk-heading-xl govuk-!-margin-top-2">Your induction programme</h1>
+    <p>Use an approved training provider</p>
+    <h2 class="govuk-heading-m">Changing your programme</h2>
+    <p>Each induction programme is setup differently. Before you change your programme:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>make sure the programme you want to change to works for your school</li>
+      <li>if you have confirmed an agreement with a training provider, you need to speak to them
```
Comment (ethanmills) [inline app/views/schools/cohorts/programme_choice.html.erb:17]: missing `</li>`

File: app/views/schools/partnerships/index.html.erb [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
   href: schools_cohort_path(@cohort.start_year))
 %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-m"><%= @cohort.start_year %></span>
-    <h1 class="govuk-heading-l">Sign up with a training provider</h1>
+    <span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
     <% if @partnership %>
-    <h2 class="govuk-heading-m">Partnership details</h2>
```
Comment (ethanmills) [inline app/views/schools/partnerships/index.html.erb:11]: I think it might be nice to split these into partials. What do you think?

**Review by ethanmills — APPROVED** [review]
Few minor things, looks good

File: app/views/schools/cohorts/programme_choice.html.erb (hunk 3 above)
Comment (amazimbe) [reply app/views/schools/cohorts/programme_choice.html.erb]: fixed

File: app/views/schools/partnerships/index.html.erb (hunk 4 above)
Comment (amazimbe) [reply app/views/schools/partnerships/index.html.erb:11]: I've done it

**Comment by amazimbe** [comment]
[Dashboard: school preparing to do a FIP](https://trello.com/c/Ij0xjt9U/487-dashboard-school-preparing-to-do-a-fip)

**Pull request merged**